.editorconfig

```
@@ -0,0 +1,12 @@
+root = true
+
+[*]
+indent_style = space
```

To match the code style:
```suggestion
indent_style = tab
```

---

crypto/tools/README.md